libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -1854,10 +1849,9 @@ string ofFilePath::getCurrentExePath(){
 }
 
 //------------------------------------------------------------------------------------------------------------
-// MARK: - near future
-//of::filesystem::path ofFilePath::getCurrentExeDir(){
-std::string ofFilePath::getCurrentExeDir(){
-	return ofFilePath::getEnclosingDirectory(ofFilePath::getCurrentExePath(), false);
+of::filesystem::path ofFilePath::getCurrentExeDir(){
```

Comment:
Not sure if this will break some previous projects because it no longer outputs a string with a trailing slash, ie:
( just showing the end part of the path, though it they both use absolute paths).
`materialPBR/bin/materialPBRDebug.app/Contents/MacOS/`
vs
`materialPBR/bin/materialPBRDebug.app/Contents/MacOS"`

Reply:
Good point. we can append preferred_separator at the end. 
and revert back to string. I thought it was more used as an internal core function.

Reply:
@dimitre Not sure how frequently it is used by externally. I don't think I have ever used it, but I just noticed that it was different and didn't want it break any previous projects :)

Reply:
I was curious myself too and I've seen it is used in different projects
https://github.com/search?q=ofFilePath%3A%3AgetCurrentExeDir%28%29&type=code